### ISSUE TITLE
configure and compiling error

### ISSUE THREAD

**Issue by bsbudhwar** [google/certificate-transparency#1167]
root@lamp /home/cts# ./configure
checking for a BSD-compatible install... /usr/bin/install -c                                                                                         
checking whether build environment is sane... yes                                                                                                    
checking for a thread-safe mkdir -p... /bin/mkdir -p                                                                                                 
checking for gawk... gawk                                                                                                                            
checking whether make sets $(MAKE)... yes                                                                                                            
checking whether make supports nested variables... yes                                                                                               
checking whether make supports nested variables... (cached) yes                                                                                      
checking for gawk... (cached) gawk                                                                                                                   
checking for g++... g++                                                                                                                              
checking whether the C++ compiler works... yes                                                                                                       
checking for C++ compiler default output file name... a.out                                                                                          
checking for suffix of executables...                                                                                                                
checking whether we are cross compiling... no                                                                                                        
checking for suffix of object files... o                                                                                                             
checking whether we are using the GNU C++ compiler... yes                                                                                            
checking whether g++ accepts -g... yes                                                                                                               
checking for style of include used by make... GNU                                                                                                    
checking dependency style of g++... gcc3                                                                                                             
checking for gcc... gcc                                                                                                                              
checking whether we are using the GNU C compiler... yes                                                                                              
checking whether gcc accepts -g... yes                                                                                                               
checking for gcc option to accept ISO C89... none needed                                                                                             
checking whether gcc understands -c and -o together... yes                                                                                           
checking dependency style of gcc... gcc3                                                                                                             
checking how to run the C preprocessor... gcc -E                                                                                                     
checking whether ln -s works... yes                                                                                                                  
checking for ranlib... ranlib                                                                                                                        
checking for ar... ar                                                                                                                                
checking the archiver (ar) interface... ar                                                                                                           
checking whether g++ supports C++11 features by default... no                                                                                        
checking whether g++ supports C++11 features with -std=gnu++11... yes                                                                                
checking for ant... ant                                                                                                                              
checking for pkg-config... /usr/bin/pkg-config                                                                                                       
checking pkg-config is at least version 0.9.0... yes                                                                                                 
checking for json_c... yes                                                                                                                           
checking for evhtp... yes                                                                                                                            
checking for icu... yes                                                                                                                              
checking for protoc... /usr/bin/protoc                                                                                                               
checking for sys/types.h... yes                                                                                                                      
checking for netinet/in.h... yes                                                                                                                     
checking for arpa/nameser.h... yes                                                                                                                   
checking for netdb.h... yes                                                                                                                          
checking for resolv.h... yes                                                                                                                         
checking how to run the C++ preprocessor... g++ -E                                                                                                   
checking for grep that handles long lines and -e... /bin/grep                                                                                        
checking for egrep... /bin/grep -E                                                                                                                   
checking for ANSI C header files... yes                                                                                                              
checking for sys/types.h... (cached) yes                                                                                                             
checking for sys/stat.h... yes                                                                                                                       
checking for stdlib.h... yes                                                                                                                         
checking for string.h... yes                                                                                                                         
checking for memory.h... yes                                                                                                                         
checking for strings.h... yes                                                                                                                        
checking for inttypes.h... yes                                                                                                                       
checking for stdint.h... yes                                                                                                                         
checking for unistd.h... yes                                                                                                                         
checking arpa/inet.h usability... yes                                                                                                                
checking arpa/inet.h presence... yes                                                                                                                 
checking for arpa/inet.h... yes                                                                                                                      
checking fcntl.h usability... yes                                                                                                                    
checking fcntl.h presence... yes                                                                                                                     
checking for fcntl.h... yes                                                                                                                          
checking limits.h usability... yes                                                                                                                   
checking limits.h presence... yes                                                                                                                    
checking for limits.h... yes                                                                                                                         
checking for netinet/in.h... (cached) yes                                                                                                            
checking stddef.h usability... yes                                                                                                                   
checking stddef.h presence... yes                                                                                                                    
checking for stddef.h... yes                                                                                                                         
checking for stdint.h... (cached) yes                                                                                                                
checking for stdlib.h... (cached) yes                                                                                                                
checking for string.h... (cached) yes                                                                                                                
checking sys/socket.h usability... yes                                                                                                               
checking sys/socket.h presence... yes                                                                                                                
checking for sys/socket.h... yes                                                                                                                     
checking sys/time.h usability... yes                                                                                                                 
checking sys/time.h presence... yes                                                                                                                  
checking for sys/time.h... yes                                                                                                                       
checking for unistd.h... (cached) yes                                                                                                                
checking leveldb/filter_policy.h usability... yes                                                                                                    
checking leveldb/filter_policy.h presence... yes                                                                                                     
checking for leveldb/filter_policy.h... yes                                                                                                          
checking event2/event.h usability... yes                                                                                                             
checking event2/event.h presence... yes                                                                                                              
checking for event2/event.h... yes                                                                                                                   
checking gflags/gflags.h usability... yes                                                                                                            
checking gflags/gflags.h presence... yes                                                                                                             
checking for gflags/gflags.h... yes                                                                                                                  
checking glog/logging.h usability... yes                                                                                                             
checking glog/logging.h presence... yes                                                                                                              
checking for glog/logging.h... yes                                                                                                                   
checking google/protobuf/message.h usability... yes                                                                                                  
checking google/protobuf/message.h presence... yes                                                                                                   
checking for google/protobuf/message.h... yes                                                                                                        
checking leveldb/db.h usability... yes                                                                                                               
checking leveldb/db.h presence... yes                                                                                                                
checking for leveldb/db.h... yes                                                                                                                     
checking ldns/ldns.h usability... yes                                                                                                                
checking ldns/ldns.h presence... yes                                                                                                                 
checking for ldns/ldns.h... yes                                                                                                                      
checking objecthash.h usability... no                                                                                                                
checking objecthash.h presence... no                                                                                                                 
checking for objecthash.h... no                                                                                                                      
checking gtest/gtest.h usability... yes                                                                                                              
checking gtest/gtest.h presence... yes                                                                                                               
checking for gtest/gtest.h... yes                                                                                                                    
checking gmock/gmock.h usability... yes                                                                                                              
checking gmock/gmock.h presence... yes                                                                                                               
checking for gmock/gmock.h... yes                                                                                                                    
checking /usr/src/gmock/gtest/src/gtest-all.cc usability... no                                                                                       
checking /usr/src/gmock/gtest/src/gtest-all.cc presence... no                                                                                        
checking for /usr/src/gmock/gtest/src/gtest-all.cc... no                                                                                             
checking /usr/src/gmock/src/gmock-all.cc usability... no                                                                                             
checking /usr/src/gmock/src/gmock-all.cc presence... yes                                                                                             
configure: WARNING: /usr/src/gmock/src/gmock-all.cc: present but cannot be compiled                                                                  
configure: WARNING: /usr/src/gmock/src/gmock-all.cc:     check for missing prerequisite headers?                                                     
configure: WARNING: /usr/src/gmock/src/gmock-all.cc: see the Autoconf documentation                                                                  
configure: WARNING: /usr/src/gmock/src/gmock-all.cc:     section "Present But Cannot Be Compiled"                                                    
configure: WARNING: /usr/src/gmock/src/gmock-all.cc: proceeding with the compiler's result                                                           
configure: WARNING:     ## -------------------------------------------------------- ##                                                               
configure: WARNING:     ## Report this to certificate-transparency@googlegroups.com ##                                                               
configure: WARNING:     ## -------------------------------------------------------- ##                                                               
checking for /usr/src/gmock/src/gmock-all.cc... no                                                                                                   
configure: error: could not find a working Google Test

**Comment by AlCutter** [comment]
Did you follow the build with gclient instructions in [README.md](https://github.com/google/certificate-transparency/blob/master/README.md)?

**Comment by bsbudhwar** [comment]
Yes i followed i tried on debian system i installed all required libraries and pre requirements also please suggest any solution.

**Comment by AlCutter** [comment]
If you're following the gclient build instructions you shouldn't be installing any required libraries, nor should you be manually running `./configure` as you've done here.

Please read the [Build with gclient](https://github.com/google/certificate-transparency/blob/master/README.md#build-with-gclient) instructions again, follow them in detail, and let us know if you still have problems.

**Comment by bsbudhwar** [comment]
Hi can i compile it on Debian my server is a debian it any issue with debian ...?
